src/main/java/Deadline.java

```diff
@@ -0,0 +1,57 @@
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+/**
+ * The {@code Deadline} class extends the {@code Task} class.
+ * Stores the date/time of the event as well.
+ */
+public class Deadline extends Task {
+
+    private String timing;
+
+    /**
+     * Initialises a Deadline task and separates the {@code desc}
+     * into the Deadline's description and date/time.
+     *
+     * @param desc Full description of the Deadline inclusive of time/date.
+     */
+    public Deadline(String desc) {
+        super(desc.split("deadline ")[1].split(" /by ")[0], "D");
```

Comment:
Maybe can extract the split part to make it easier to read?

Reply:
LGTM, I'll see how I can improve readability. Thanks.